### PR TITLE
Make entry comments popup as per mockup

### PR DESCRIPTION
**Pull request by subinasr**
- Addresses #4254

## Changes

* Make entries comments popup as per the mockup

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

File: app/components/entryReview/EntryComments/CommentForm/index.tsx [hunk 1]
```diff
@@ -121,21 +121,25 @@ function CommentForm(props: Props) {
             >
                 <TextArea
                     name="text"
+                    className={styles.commentArea}
                     rows={3}
                     value={value.text}
                     onChange={setFieldValue}
                     error={error?.text}
                     autoFocus
+                    // TODO: remove the underlining border of this text input
```
Comment (tnagorra) [inline app/components/entryReview/EntryComments/CommentForm/index.tsx]: Let's create variants for Inputs

File: app/components/entryReview/EntryComments/Comment/index.tsx [hunk 2]
```diff
@@ -62,69 +62,60 @@ function Comment(props: Props) {
...
-                <EditCommentForm
-                    className={styles.comment}
-                    comment={comment}
-                    onEditSuccess={handleSuccess}
-                    onEditCancel={hideEditModal}
+            <>
+                <DateOutput
+                    className={styles.date}
+                    value={createdAt}
+                    format="MMM dd, yyyy . hh:mm aaa"
```
Comment (tnagorra) [inline app/components/entryReview/EntryComments/Comment/index.tsx]: Let's revert this change

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/components/entryReview/EntryComments/CommentForm/index.tsx [hunk 3]
```diff
@@ -121,21 +121,26 @@ function CommentForm(props: Props) {
...
                     name="mentionedUsers"
+                    inputSectionClassName={styles.selectInput}
                     label="Assignees"
                     value={value.mentionedUsers}
                     projectId={projectId}
                     onChange={setFieldValue}
                     options={members}
                     onOptionsChange={setMembers}
                     error={getErrorString(error?.mentionedUsers)}
+                    variant="general"
```
Comment (tnagorra) [inline app/components/entryReview/EntryComments/CommentForm/index.tsx]: What does this variant do?

**Review by tnagorra — APPROVED** [review]

File: app/components/entryReview/EntryComments/CommentForm/index.tsx (hunk 3 above)
Comment (tnagorra) [reply app/components/entryReview/EntryComments/CommentForm/index.tsx]: @subinasr ?

Comment (subinasr) [reply app/components/entryReview/EntryComments/CommentForm/index.tsx]: It WAS for the styling purpose, but it's removed now.
